src/bootstrap/user.go

```diff
@@ -0,0 +1,103 @@
+package bootstrap
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"github.com/syndtr/goleveldb/leveldb"
+	"golang.org/x/crypto/bcrypt"
+	"gorm.io/driver/sqlite"
+	"gorm.io/gorm"
+	"log"
+	"math/rand"
+	"os"
+)
+
+type User struct {
+	gorm.Model
+	Username string `json:"username",gorm:"uniqueIndex,not null"`
+	Password string `json:"password",gorm:"not null"`
+	Role     string `json:"role",gorm:"not null"`
+	Email    string `json:"email"`
+}
+
+func MigrateLevelDBToSqlite(oldDBFile, newDBFile string) {
+	oldDB, err := leveldb.OpenFile(oldDBFile, nil)
+	if err != nil {
+		panic(err)
+	}
+
+	newDB, err := gorm.Open(sqlite.Open(newDBFile), nil)
```

Comment:
Might need to defer closing here as well?

Reply:
Closing here, yes. Closing it in auth.go is not good, cause we only need the connection ones and hold it until the program closes.

Reply:
Right that makes sense.